include/game.h

```diff
@@ -1796,7 +1796,7 @@ typedef struct {
     /* 80072F3C */ s32 unk1C;
     /* 80072F40 */ s32 unk20;
     /* 80072F44 */ s32 unk24;
-    /* 80072F48 */ s32 unk28;
+    /* 80072F48 */ void (*unk28)(Entity*);
```

Comment:
Any reason not to use the `PfnEntityUpdate` type? `func_psp_0923B2F0` looks like a typical entity state machine function.

Reply:
No reason, just copied out of the old definition https://github.com/Xeeynamo/sotn-decomp/pull/2247/files#diff-4dbceb2f69232a451b571009bbb93e8c561300b5c08d314cafe8a39f5c49d6d2L1932

Thanks for the feedback, have updated